### ISSUE TITLE
Notebooks are not tested during CI

### ISSUE THREAD

**Issue by bdestombe** [ArtesiaWater/nlmod#41]
I thought this was the case before. Please have a look at the dtscalibration package if you experience any issues with the implementation.

**Comment by OnnoEbbens** [comment]
Yes I removed these tests because they took so long. They are explicitly excluded in ci.yml using `py.test ./tests -m "not notebooks"`.

We can reinstate them but I think we should first improve test efficiency (see also #21).

**Comment by bdestombe** [comment]
Ah I see. What about the following two options?
 - start two CI pipelines in parallel. One with the notebooks and one with the other tests. While developing only look at the latter one and requiring both for a version bump. I recon it is even worthwhile paying for two CI sessions.
 - Moving the notebook tests to the end of the CI.

**Comment by bdestombe** [comment]
Onno and I discussed over the phone that I would look into the possibilities of having two parallel pipelines

**Comment by OnnoEbbens** [comment]
Solve together with #21

**Comment by rubencalje** [comment]
The notebooks are not tested directly, but readthedocs does run then before publishing them on nlmod.readthedocs.io, where its output can be checked.
